test/cmdlineTests/yul_optimize_runs/output

```diff
@@ -13,15 +13,15 @@ object "RunsTest1" {
     object "Runtime" {
         code {
             {
-                mstore(0, 0xabc1234500000000000000000000000000000000000000000000000000000000)
+                sstore(0, 0xabc1234500000000000000000000000000000000000000000000000000000000)
```

Comment:
So you changed this manually to create a different test?

Reply:
Yes.